src/Compiler/TypedTree/TypedTree.fs

```diff
@@ -5294,13 +5295,13 @@ type CcuData =
 type CcuTypeForwarderTree<'TKey, 'TValue> =
```

Comment:
Just a comment that either 

1. Make this a generally-named thing in Utilities ("LookupTree") OR
2. Key and Value should be specialized to their specific types `string` and `Lazy<EntityRef>>`.  

At the moment the name is specific but the code is generic (and only ever instantiated at one type), which doesn't sit quite right